### ISSUE TITLE
Flightplan line disappearing from the Navigation Display

### ISSUE THREAD

**Issue by ErikNL80** [flybywiresim/a32nx#5178]
### Aircraft Version

Development

### Build info

```json
{
  "built": "2021-06-20T23:50:13+00:00",
  "ref": "master",
  "sha": "85470d645814125ff79304bfd19a438e721c736d",
  "actor": "MisterChocker",
  "event_name": "manual"
}
```

### Describe the bug

Today did a flight from EGNS (Isle of Man) to Faro (LPFR). I used the built-in flight planner to plan the flight. For arrival I only selected the ILS28 approach, no further STAR. When I arrived near Faro the latest waypoint shown is GEBTI and from there the line disappears. When coming closer, the entire green line disappeared. I also attached a shot from the MCDU, where the waypoints are shown. 

The plane is following the flightplan correctly, it is only a visual issue on the ND.
![LPFR](https://user-images.githubusercontent.com/71258178/122749671-e3436a80-d28d-11eb-8ad4-a80493099c18.png)
![LPFR2](https://user-images.githubusercontent.com/71258178/122749683-e76f8800-d28d-11eb-8f11-3c4864897a40.png)
![LPFR3](https://user-images.githubusercontent.com/71258178/122749693-eb030f00-d28d-11eb-80f2-8318e5f52d45.png)

### Expected behavior

I would expect the green line to follow the entire flightplan including SID/STARS from airport to airport.

### Steps to reproduce

1. Create a flightplan using the flightplanner
2. Follow the route and check for inconsistensies during take-off and landing

### References (optional)

Screenshots attached above.

**Comment by roxter69** [comment]
just did a flight from faroe islands to tromso 300 nm to go flight plan dissapears,im also using the in game flight planner,and also the latest dev version,this has happened on and off randomly for a while now

**Comment by Road-to-Hell9** [comment]
Same here Version: 2a23f44
![Microsoft Flight Simulator Screenshot 2021 07 02 - 02 44 18 54](https://user-images.githubusercontent.com/69807128/124204484-775bd000-dadf-11eb-8825-a45de31a2a87.png)

**Comment by Benjozork** [comment]
This will be fixed in the custom FMGC. Please keep comments relevant to new information about the issue.

**Comment by 2hwk** [comment]
Fixed by #4953 closing ☑️
